File: Java/EjemploAbm/src/modelo/UsuarioDao.java
package modelo;

import java.sql.ResultSet;
import java.sql.SQLException;

import org.apache.commons.codec.digest.DigestUtils;

public class UsuarioDao {
	
	
	
	public Boolean login(String usuario, String pass) {
		DataBase conexion = DataBase.getInstance();
		String query = "SELECT * FROM usuario WHERE usuario = '"+usuario+"' AND password = '"+DigestUtils.md5Hex(pass)+"'";
		ResultSet result = conexion.query(query);
		try {
			if (result.next()) {
				return true;
			}else return false;
		} catch (SQLException e) {
			e.printStackTrace();
			return false;
		}
	}
	
	public Boolean agregar(String nombre, String apellido, String usuario, String pass) {
		DataBase conexion = DataBase.getInstance();
		String query = "INSERT INTO public.usuario(nombre, apellido, password, usuario) VALUES ( '"+nombre+"', '"+apellido+"', '"+DigestUtils.md5Hex(pass)+"', '"+usuario+"')";
		Boolean result = conexion.insert(query);
		System.out.println(result);
		return result;
		
	}

}

File: Python/EjemploAbm/modelo/UsuarioDao.py
from modelo.bd import Database
import hashlib

class UsuarioDao:
    
    def __init__(self):
        self.__bd = Database()
    
    def login(self, usuario, passw):
        md5 = hashlib.new("md5")
        md5.update(passw.encode())
        result = self.__bd.get("SELECT * FROM usuario WHERE usuario = '{}' AND password = '{}'".format(usuario,md5.hexdigest()))
        if result:
            return True
        return False

    def get_all(self):
        return self.__bd.getAll("SELECT nombre,apellido, usuario FROM usuario")

    def insert(self,nombre, apellido, usuario, passw):
        md5 = hashlib.new("md5")
        md5.update(passw.encode())
        return self.__bd.query("INSERT INTO public.usuario(nombre, apellido, password, usuario) VALUES ( '{}', '{}', '{}', '{}');".format(nombre, apellido,md5.hexdigest(),usuario))